### PR TITLE
Python 3 Django 1.11 (issue 22)

### PR DESCRIPTION
**Pull request by guy881**
https://github.com/IMGIITRoorkee/django-filemanager/issues/22

- Tested manually on Python 3.6 and Django 1.8 & 1.11
- I couldn't got a working CK Editor widget, so I couldn't test it, my attempt is on different branch [test-widget](https://github.com/guy881/django-filemanager/tree/test-widget).
- I added a really simple unit test (FilemanagerTest), but runtests.py seems to not detect this one. I was running it directly from IDE, so it wasn't a problem for me, but maybe we can improve this. I would like to write more tests to make this project more maintainable.

### CODE REVIEW

**Comment by akki** [comment]
Hey @guy881 
Thank you so much for taking this forward. I was able to have a quick look and this looks like a good start for me.

One place that I felt still needs some help were the [docs](https://github.com/IMGIITRoorkee/django-filemanager#as-a-filemanager).
Also, is this code still Python2 compatible? If not, it might be worth documenting this detail somewhere IMO.

**Comment by guy881** [comment]
Hello @akki 
Thank you for your review, I am glad you liked it, I've made some updates to docs, as you suggested. Yes, this is still compatible with Python 2, I just pointed it in docs, to be clear.

**Comment by guy881** [comment]
@akki can you please have a look on this one? I really want this to be merged, as it is a part of my university project. If there is anything else I should improve here, please let me know.

**Comment by akki** [comment]
Hi @guy881 Thank you for pinging me again (I had missed your last ping due to holiday season, sorry).

I think this is in better shape now. Also appreciate the improved formatting in the docs.

I tried running the code manually and found one little change required that I have pointed out above.
I think this is good to be merged if you could make that change and squash your commits into one with a good commit message.

Also, as you mentioned earlier if you could add some testcases that would be really awesome and helpful - maybe in this PR or you can open another one (I leave that choice upto you).

File: README.md [hunk 1]
```diff
@@ -18,77 +24,88 @@ Install it by
...
 ----------------
 
-In urls.py of your app to make filemanager run at url /abc/
-<pre>
+In `urls.py` of your app to make filemanager run at url `/abc/`.
+```python
+from django.conf.urls import url
+
 from filemanager import path_end
 from views import view
```
[the remark this answers is not in the document]
Comment (guy881) [reply README.md]: Thank you for suggestion @akki.
For me `from views import view` is working. I guess we're running the app differently. How should i run it?
If I run it by `python tests/manage.py runserver` I am getting an error: `ModuleNotFoundError: No module named 'filemanager'`

Right now I am using PyCharm with option to add directories to PYTHONPATH, so I guess this is the difference.

[the remark this answers is not in the document]
Comment (akki) [reply README.md]: I think it is fine in that case.

**Comment by akki** [comment]
Thanks @guy881 - this is in master now as c161f30.

**Comment by guy881** [comment]
Thanks a lot @akki!

**Pull request merged**